pages/common/mongoimport.md

```diff
@@ -0,0 +1,24 @@
+# mongoimport
+
+> Imports content from a JSON, CSV, or TSV file into a MongoDB.
+> More information: <https://docs.mongodb.com/database-tools/mongoimport/>.
+
+- Import a JSON file into a specific collection:
+
+`mongoimport --file={{path/to/file}} --uri={{mongodb_uri}} --collection={{collection_name}}`
```

Comment:
Shouldn't that be `path/to/file.json`?

Reply:
The extension is not really mandatory. It could be a custom one too and the process won't be affected.